### PR TITLE
zh-translation: docs/tasks/security/authorization/authz-td-migration/index.md

### PR DESCRIPTION
**Pull request by Ruil1n**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
/ok-to-test

File: content/zh/docs/tasks/security/authorization/authz-td-migration/index.md [hunk 1]
```diff
@@ -95,31 +86,16 @@ test: yes
...
-          trustDomain: new-td
-    EOF
-    $ istioctl manifest apply --set profile=demo -f td-installation.yaml
+    $ istioctl install --set profile=demo --set meshConfig.trustDomain=new-td
     {{< /text >}}
 
-    Istio 网格现在运行于一个新的信任域 `new-td` 了。
-
-1. 删除 `default` 和 `sleep-allow` 命名空间中的 `sleep` 和 `httpbin` 的 secrets。请注意如果您安装 Istio 时启用了 SDS，您可以跳过这一步。
+1. 重新部署istiod以使信任域更改。
```
Comment (MXuDong) [inline content/zh/docs/tasks/security/authorization/authz-td-migration/index.md:92]:
```suggestion
1. 重新部署 istiod 以使信任域更改。
```

File: content/zh/docs/tasks/security/authorization/authz-td-migration/index.md [hunk 2]
```diff
@@ -16,23 +16,14 @@ test: yes
 
 ## 开始之前{#before-you-begin}
 
-1. 阅读[授权概念指南](/zh/docs/concepts/security/#authorization)。
+在您开始任务之前，请完成以下内容。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-td-migration/index.md:19]:
```suggestion
在您开始任务之前，请完成以下内容：
```

File: content/zh/docs/tasks/security/authorization/authz-td-migration/index.md [hunk 3]
```diff
@@ -16,23 +16,14 @@ test: yes
...
+
+1. 阅读[Istio 授权概念指南](/zh/docs/concepts/security/#authorization)。
```
Comment (kebe7jun) [inline content/zh/docs/tasks/security/authorization/authz-td-migration/index.md:21]: 与中文标题保持一致。
```suggestion
1. 阅读 [授权](/zh/docs/concepts/security/#authorization) 指南。
```

**Review by kebe7jun — APPROVED** [review]
lgtm

**Pull request merged**